server/belga/search_providers.py

```diff
@@ -332,7 +335,7 @@ def _get_body_html(self, item):
         return body.replace('\n', '<br/>&nbsp;&nbsp;&nbsp;&nbsp;')
 
     def _get_abstract(self, item):
-        return self._get_newscomponent(item, 'lead')
+        return '&nbsp;&nbsp;&nbsp;&nbsp;' + get_text(self._get_newscomponent(item, 'lead')) + '<br/><br/>'
```

Comment:
why those spaces in front?

Reply:
so that it looks similar to the way we have body_html on UI.

Reply:
and we do it when we create content? or when we ingest something? I thought that's just css

Reply:
here you can see they have added the spaces in front of body HTML https://github.com/superdesk/superdesk-belga/blob/ed02e586f1a36c0bf0044e5c14c95d1d2298d800/server/belga/search_providers.py#L334 so I have spaces for abstract also so that both looks good and the same.